### ISSUE TITLE
this.console_error is not a function

### ISSUE THREAD

**Issue by roflcoopter** [gadgetchnnel/lovelace-card-preloader#6]
I am getting this error in Chrome dekstop on the latest 0.0.4
```
lovelace-card-preloader.js:1 Uncaught (in promise) TypeError: this.console_error is not a function
    at console.window.console.error (lovelace-card-preloader.js:1)
    at l (chunk.8b29f0c82629b54e915d.js:12852)
    at Object.Kt [as createCardElement] (chunk.8b29f0c82629b54e915d.js:12311)
    at Object.preloadElement (lovelace-card-preloader.js:1)
    at lovelace-card-preloader.js:1
    at Array.forEach (<anonymous>)
    at lovelace-card-preloader.js:1

**Comment by gadgetchnnel** [comment]
This should be fixed in the latest version, 0.0.5, can you please try it and see if it works for you?

**Comment by roflcoopter** [comment]
Error is gone, thanks a lot!
